java/server/src/org/openqa/grid/web/servlet/HubStatusServlet.java

```diff
@@ -167,4 +185,57 @@ protected void process(
       throw new IOException(e);
     }
   }
+
+  private static boolean IsKeyPresentIn(List<String> keys, String key) {
```

Comment:
Nit: java method names begin with a lower-case letter. We can fix this once the PR is merged in.